sockstat: split FreeBSD, NetBSD and Linux implementations (#12040)
tldr-pages/tldr: pages/freebsd/sockstat.md

# sockstat

> List open Internet or UNIX domain sockets.

> > Some of the displayed informations: USER, COMMAND, PID, FD (file descriptor), PROTO (protocol).

Reviewer: ```suggestion
```

> More information: <https://man.freebsd.org/cgi/man.cgi?sockstat>.

> - View which users/processes are [l]istening to which ports:

Reviewer 2: ```suggestion
- View which users/processes are [l]istening on which ports:
```

`sockstat -l`

- Show information for IPv[4]/IPv[6] sockets [l]istening on specific [p]orts using a specific [P]rotocol:

`sockstat -{{4|6}} -l -P {{tcp|udp|sctp|divert}} -p {{port1,port2...}}`

> - Also show [c]onnected sockets, not resolving numeric UIDs to user [n]ames and using a [w]ider field size:

Reviewer 2: ```suggestion
- Also show [c]onnected sockets, not resolving [n]umeric UIDs to user names and using a [w]ider field size:
```
